### PR TITLE
SEO- 176062 windows forms duplicate titles

### PR DESCRIPTION
**Pull request by Edithabich**
HI @MallikaRK ,
Have updated UG titles please review.
Thanks.

Regards,
Edith Abich.

### CODE REVIEW

File: WindowsForms/ComboBox/Overview.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: About Windows Forms ComboBox control | Syncfusion
+title: Overview of Windows Forms ComboBox control | Syncfusion
```
Comment (Yvone-Atieno) [inline WindowsForms/ComboBox/Overview.md:3]: Control, C should be in caps. Check in all places

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: WindowsForms/Form/Overview.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: About Windows Forms Form control | Syncfusion
+title: Overview of Windows Forms Control | Syncfusion
```
Comment (Yvone-Atieno) [inline WindowsForms/Form/Overview.md:3]: Overview of Windows Forms Form Control | Syncfusion

File: WindowsForms/layoutmanagers/BorderLayout/Overview.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: About Windows Forms LayoutManagers control | Syncfusion
+title: Overview of Windows Forms BorderLayout Control | Syncfusion
```
Comment (Yvone-Atieno) [inline WindowsForms/layoutmanagers/BorderLayout/Overview.md:3]: Overview of Windows Forms LayoutManagers Control | Syncfusion

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: WindowsForms/Form/Overview.md (hunk 2 above)
Comment (Edithabich) [reply WindowsForms/Form/Overview.md]: resolved

File: WindowsForms/layoutmanagers/BorderLayout/Overview.md (hunk 3 above)
Comment (Edithabich) [reply WindowsForms/layoutmanagers/BorderLayout/Overview.md]: resolved

**Pull request closed without merging**